internal/cli/atlas/livemigrations/validation/create.go

```diff
@@ -0,0 +1,187 @@
+	cli.GlobalOpts
+	cli.OutputOpts
+	destinationClusterName      string
+	destinationDropEnabled      bool
+	migrationHosts              []string
+	sourceCACertificatePath     string
+	sourceClusterName           string
+	sourceProjectID             string
+	sourceSSL                   bool
+	sourceManagedAuthentication bool
+	sourceUsername              string
+	sourcePassword              string
+	store                       store.LiveMigrationValidationsCreator
+}
+
+func (opts *CreateOpts) initStore() error {
+	var err error
+	opts.store, err = store.New(store.AuthenticatedPreset(config.Default()))
+	return err
+}
+
+var createTemplate = `ID	PROJECT ID	SOURCE PROJECT ID	STATUS
+{{.ID}}	{{.GroupID}}	{{.SourceGroupID}}	{{.Status}}`
+
+func (opts *CreateOpts) Run() error {
+	if err := opts.askDestinationDropConfirm(); err != nil {
+		return err
+	}
+
+	if err := opts.askPassword(); err != nil {
+		return err
+	}
+
+	createRequest := opts.newValidationCreateRequest()
+
+	r, err := opts.store.CreateValidation(opts.ConfigProjectID(), createRequest)
+	if err != nil {
+		return err
+	}
+
+	return opts.Print(r)
+}
+
+func (opts *CreateOpts) newValidationCreateRequest() *mongodbatlas.LiveMigration {
+	return &mongodbatlas.LiveMigration{
+		Source: &mongodbatlas.Source{
+			ClusterName:           opts.sourceClusterName,
+			GroupID:               opts.sourceProjectID,
+			Username:              opts.sourceUsername,
+			Password:              opts.sourcePassword,
+			SSL:                   &opts.sourceSSL,
+			CACertificatePath:     opts.sourceCACertificatePath,
+			ManagedAuthentication: &opts.sourceManagedAuthentication,
+		},
+		Destination: &mongodbatlas.Destination{
+			ClusterName: opts.destinationClusterName,
+			GroupID:     opts.ConfigProjectID(),
+		},
+		MigrationHosts: opts.migrationHosts,
+		DropEnabled:    &opts.destinationDropEnabled,
+	}
+}
+
+func (opts *CreateOpts) askDestinationDropConfirm() error {
+	if !opts.destinationDropEnabled {
+		return nil
+	}
+	confirmDrop := false
+	p := prompt.NewConfirm("are you sure you want to drop the destination collections?")
+
+	if err := survey.AskOne(p, &confirmDrop); err != nil {
+		return err
+	}
+
+	if !confirmDrop {
+		return errors.New("user-aborted. Not dropping destination collections")
+	}
+	return nil
+}
+
+func (opts *CreateOpts) askPassword() error {
+	if opts.sourceManagedAuthentication {
+		return nil
+	}
+	if opts.sourcePassword != "" {
+		return nil
+	}
+	p := &survey.Password{
+		Message: "Password:",
+	}
+	if err := survey.AskOne(p, &opts.sourcePassword); err != nil {
+		return err
+	}
+	if opts.sourcePassword == "" {
+		return errors.New("no password provided")
+	}
+	return nil
+}
+
+func (opts *CreateOpts) validate() error {
+	if opts.sourceManagedAuthentication && opts.sourceUsername != "" {
+		return fmt.Errorf("--%s=%t and --%s are exclusive", flag.LiveMigrationSourceManagedAuthentication, opts.sourceManagedAuthentication, flag.LiveMigrationSourceUsername)
```

Comment:
[super-nit] we try to avoid references to the format `--%s=%t`, in particular the flag`equals`value format, we got reports of users confused by it and specially on boolean flags because `--drop=false` is valid but `--drop false` is not so if we can let's not use it here either

Reply:
Cool, how about `--sourceManagedAuthentication set to true and --sourceUsername are exclusive`?

```suggestion
		return fmt.Errorf("--%s=%t and --%s are exclusive", flag.LiveMigrationSourceManagedAuthentication, opts.sourceManagedAuthentication, flag.LiveMigrationSourceUsername)
```

Reply:
> --sourceManagedAuthentication is set and can't use --sourceUsername 

Or something, boolean flags are only true (unless using --sourceManagedAuthentication=false but that's already the default value)

Reply:
Updated:
```
bin/mongocli atlas lm validation create --destinationClusterName=Cluster0 --migrationHost=colmqmac --sourceClusterName=myReplicaSet --sourceProjectId=611e68cb086adf5cda2e6253 --profile=dev --sourceManagedAuthentication --sourceUsername=test
Error: --sourceManagedAuthentication and --sourceUsername are exclusive
```

Reply:
Also for the username example:
```
bin/mongocli atlas lm validation create --destinationClusterName=Cluster0 --migrationHost=colmqmac --sourceClusterName=myReplicaSet --sourceProjectId=611e68cb086adf5cda2e6253 --profile=dev
Error: MongoDB Automation is not managing authentication, --sourceUsername must be set
```